interface.go

```diff
@@ -223,6 +223,14 @@ type Config struct {
 	// If the timeout is exceeded, the connection is closed.
 	// If this value is zero, the timeout is set to 30 seconds.
 	IdleTimeout time.Duration
+	// AttackTimeout is the maximum duration that may pass after a suspicious packet is
+	// received without any incoming network activity, used for experimental mitigation
+	// against injection attacks. If the timeout is exceeded, the connection is closed.
+	// This timeout should be longer than any reasonable round trip time, but excessively
+	// long timeouts will delay reporting of rejected connections.
+	// If this value is zero, the default mitigation behavior will be used (currently off).
+	// If this value is negative, mitigation will be disabled.
+	AttackTimeout time.Duration
```

Comment:
Thinking: Maybe it would be more useful to define this in terms of RTTs. That would also make it easier to switch this to a bool, since we could then pick a reasonable default value.

Unless of course an attacker could influence our RTT estimate. Is that the case?

Reply:
It's safest to assume that the attacker has control over the RTT estimate until we get the 1-RTT keys. That said, there may be a more intuitive way to represent this AttackTimeout.